net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/BgpController.php

```diff
@@ -2,6 +2,7 @@
 namespace OPNsense\Quagga;
 /*
     Copyright (C) 2017 Fabian Franz
+    Modified for BGP functionality by Michael Muenz
```

Comment:
It should really read the following:

Copyright (C) 2017 Michael Muenz

don't be shy :)